### PR TITLE
Add BitcoinSServerMain test to connect to bitcoind

### PR DESCRIPTION
**Pull request by Christewart**

### CODE REVIEW

**Comment by Christewart** [comment]
Compile failing: https://github.com/bitcoin-s/bitcoin-s/pull/3591/checks?check_run_id=3413500240#step:5:189

**Comment by benthecarman** [comment]
CI failing is from flaky node tests

**Comment by Christewart** [comment]
> CI failing is from flaky node tests

It looks like there is a failure with the new tests: https://github.com/bitcoin-s/bitcoin-s/pull/3591/checks?check_run_id=3421522083#step:5:3615

File: testkit/src/main/scala/org/bitcoins/testkit/fixtures/BitcoinSAppConfigFixture.scala [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import org.bitcoins.testkit.{BitcoinSTestAppConfig, EmbeddedPg}
+import org.scalatest.FutureOutcome
+
+import scala.concurrent.Future
+
+sealed trait BitcoinSAppConfigFixture extends BitcoinSFixture with EmbeddedPg {
+  override type FixtureParam = BitcoinSAppConfig
+
+  override def afterAll(): Unit = {
+    println(s"Stopping BitcoinSAppConfigFixture")
```
Comment (Christewart) [inline testkit/src/main/scala/org/bitcoins/testkit/fixtures/BitcoinSAppConfigFixture.scala]: remove, i think i added this. Sorry about that.

**Review by benthecarman — APPROVED** [review]

**Pull request merged**